notebooks/1_spikesorting.ipynb

```diff
@@ -4,26 +4,16 @@
    "cell_type": "markdown",
```

Comment:
<p>This is neat but is it necessary for spike sorting? I think the notebook should be focused on showing spike sorting and only that. Otherwise distracting.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/LorenFrankLab/nwb_datajoint/pull/109/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>